hypershift-operator/controllers/nodepool/metrics/metrics.go

```diff
@@ -31,8 +31,8 @@ const (
 	CountByHClusterMetricName = "hypershift_hostedcluster_nodepools" // What about renaming it to hypershift_cluster_nodepools ?
 	countByHClusterMetricHelp = "Number of NodePools for a given HostedCluster"
 
-	CoresCountByHClusterMetricName = "hypershift_cluster_cores"
-	CoresCountByHClusterMetricHelp = "Number of cores for a given HostedCluster. " +
+	VCpusCountByHClusterMetricName = "hypershift_cluster_vcpus"
+	VCpusCountByHClusterMetricHelp = "Number of vCpus for a given HostedCluster. " +
```

Comment:
vCPUs is a very loaded term. It would be good to clarify exactly what it means (e.g. "number of virtual CPUs as reported by the platform").
@kahowell (since you're going to leverage this metric) do you want to chime in?

Reply:
Agreed, I will change the description a bit to be more explicit.

Reply:
:+1: on "as reported by the platform". It might also be useful to point out that this is a measure of vCPU capacity available to customer workloads.